net.wz2100.wz2100.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+	<id>net.wz2100.wz2100</id>
+	<metadata_license>gpl-2</metadata_license>
+	<summary>In Warzone 2100, you command the forces of The Project in a battle to rebuild the world after mankind has almost been destroyed by nuclear missiles.</summary>
+	<description>
+	<p>
+	In Warzone 2100, you command the forces of The Project in a battle to rebuild the world after mankind has almost been destroyed by nuclear missiles.
+	</p>
+	The game offers campaign, multi-player, and single-player skirmish modes. An extensive tech tree with over 400 different technologies, combined with the unit design system, allows for a wide variety of possible units and tactics.
+	<p>
+	Warzone 2100 was originally developed as a commercial game by Pumpkin Studios and published in 1999, and was released as open source by them in 2004, for the community to continue working on it.
+	</p>
+	</description>
+	<url type="homepage">http://wz2100.net/</url>
+	<screenshots>
+		<screenshot type="default">
+		<image>http://wz2100.net/static/img/wz2100-CAM_3A50.png</image>
+		</screenshot>
+	</screenshots>
+	<project_license>GPL-2.0+</project_license>
```

Comment:
AFAIK it's no longer correct and should be GPL-2.0-or-later.

Reply:
I believe our latest info is that both are OK in the ecosystem (as opposed to the SPDX spec), so I've stopped suggesting that now.

Reply:
:+1: